components/brave_rewards/browser/publisher_info_database_unittest.cc

```diff
@@ -645,7 +646,7 @@ TEST_F(PublisherInfoDatabaseTest, GetPanelPublisher) {
   filter_4.id = "page.com";
   filter_4.reconcile_stamp = 10;
   ledger::PublisherInfoPtr result =
-      publisher_info_database_->GetPanelPublisher(filter_4);
+      publisher_info_database_->GetPanelPublisher(filter_4.Clone());
```

Comment:
`filter_4.Clone()` should use `std::move(filter_4)` as `filter_4` is no longer used